extensions/scripts/run_chrome.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
Can we have this dir as `script` (singular)? This is to match the default rails project structure, where this folder is named in singular.